.github/workflows/ci.yml

```diff
@@ -27,7 +27,8 @@ jobs:
         run: |
           brew install cmake
           brew install libomp
-          cargo build --all-features
+          brew link --force libomp
```

Comment:
does this do anything? It seems a bit strange to run link right after install

Reply:
When it failed to compile using the beta toolchain in CI, it didn't find libomp, that's why we force link it here, to avoid something like that happening.